.github/workflows/crucible-ci.yaml

```diff
@@ -13,7 +13,35 @@ on:
   workflow_dispatch:
 
 jobs:
+  crucible-ci-check-for-controller-change:
+    runs-on: ubuntu-latest
+
+    outputs:
+      build_controller: ${{ steps.change-check.outputs.answer }}
+
+    steps:
+    - name: Checkout crucible
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
+
+    - name: Check for changes in the workshop directory
+      id: change-check
+      run: |
+        diff_cmd="git diff --name-only HEAD^1 HEAD"
+        echo "Files changed:"
+        ${diff_cmd}
+        echo
+        workshop_files_changed=$(${diff_cmd} | grep "^workshop/" | wc -l)
+        echo "workshop_files_changed=${workshop_files_changed}"
+        echo
+        if [ ${workshop_files_changed} -gt 0 ]; then answer="yes"; else answer="no"; fi
+        echo "Setting answer to '${answer}'"
+        echo "::set-output name=answer::${answer}"
```

Comment:
have you seen this ? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths

This seems to do the same as you are doing manually...

Reply:
I'm already using that in this workflow.  What I was trying to do here was avoid having to create yet another workflow that only had a single parameter difference from this one.  By adding this job and creating an output from it that it used as input to the other jobs I was able to keep a single workflow with all the other logic consolidated.

Reply:
oh I see, any reason you want one single workflow ?

Reply:
To avoid replication and having to keep the workflows synchronized.

Reply:
simplicity, fair enough